### DISCUSSION TITLE
Update "Setting up the development environment" documentation to recommend using a Ruby Version Manager after the introduction of Gemfile and .ruby-version

### DISCUSSION THREAD

**Discussion by Adnan-Bacic** [react-native-community/discussions-and-proposals#454] [Ideas]
Recently this proposal was accepted: https://github.com/react-native-community/discussions-and-proposals/discussions/411

and version 0.67 of react-native comes with a `Gemfile` and a `.ruby-version`.

Inside the Gemfile there is a comment saying:
`# You may use http://rbenv.org/ or https://rvm.io/ to install and use this version`

However the documentation for [environment-setup](https://reactnative.dev/docs/environment-setup) says this:
`You can use a Ruby Version manager, however we recommend that you use the standard Ruby available on macOS unless you know what you're doing.`

I feel like these contradict each other. The documentation recommends not using a Ruby Version Manager while the project files do.

So i would propose to update the documentation to recommend using a Ruby Version Manager, similarly to the `Gemfile`.

**Comment by cortinico** [comment]
> I feel like these contradict each other.

I don't think they actually contradict each other. The template is "ready to work" with a Ruby Version manager. However, we recommend you use the standard Ruby on macOS, mostly because we want to reduce the number of steps needed to setup a new RN project.

**Reply by Adnan-Bacic** [reply]
i see the reason for that. though i did just notice that the documentation for windows + android has this line:

>It is recommended to use an LTS version of Node. If you want to be able to switch between different versions, you might want to install Node via nvm-windows, a Node version manager for Windows.

macOS just recommends using brew and not nvm.

maybe instead of explicitly recommending using a Ruby Version Manager, it could similarly just suggest using it. so people are aware it exists.

**Reply by cortinico** [reply]
Yes, all the platforms should all be aligned to suggest a version manager, that's reasonable (both for node and ruby).
